app/frontend/src/features/communities/CommunityPage/PlaceCard.tsx

```diff
@@ -40,26 +50,36 @@ export default function PlaceCard({
     [place.content]
   );
   return (
-    <Card className={classNames(classes.root, className)}>
-      <CardActionArea>
-        <CardMedia
-          image="https://loremflickr.com/320/240"
-          className={classes.image}
-        />
+    <Card className={className}>
+      <Link
+        to={routeToPlace(place.pageId, place.slug)}
+        className={classes.link}
+        component={CardActionArea}
+      >
+        <CardMedia src={place.photoUrl} className={classes.image} />
```

Comment:
Getting this runtime warning in the console from MUI:

```
Warning: Failed prop type: Material-UI: Either `children`, `image`, `src` or `component` prop must be specified.
```

Might be worth falling back to some stock image if `photoUrl` is undefined?